### PR TITLE
Decrease exec frequency to weekly for 4.10 periodics

### PR DESCRIPTION
**Pull request by jmguzik**
/cc @jupierce 
/cc @sdodson
Signed-off-by: Jakub Guzik <jguzik@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/cluster-api-provider-ovirt/openshift-cluster-api-provider-ovirt-release-4.10.yaml [hunk 1]
```diff
@@ -50,13 +50,13 @@ tests:
     cluster_profile: ovirt
     workflow: openshift-e2e-ovirt-minimal
 - as: e2e-ovirt-cleanup
-  interval: 4h
+  cron: 48 22 */7 * *
   steps:
     cluster_profile: ovirt
     test:
     - ref: openshift-e2e-ovirt-cleanup-test
 - as: e2e-ovirt-prfinder
```
Comment (jmguzik) [inline ci-operator/config/openshift/cluster-api-provider-ovirt/openshift-cluster-api-provider-ovirt-release-4.10.yaml:58]: suspicious

File: ci-operator/config/openshift/cluster-api-provider-ovirt/openshift-cluster-api-provider-ovirt-release-4.10.yaml [hunk 2]
```diff
@@ -50,13 +50,13 @@ tests:
...
-  interval: 5m
```
Comment (jmguzik) [inline ci-operator/config/openshift/cluster-api-provider-ovirt/openshift-cluster-api-provider-ovirt-release-4.10.yaml:59]: as well

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml [hunk 3]
```diff
@@ -29,25 +29,25 @@ tests:
...
     cluster_profile: gcp-openshift-gce-devel-ci-2
     workflow: openshift-upgrade-gcp
 - as: e2e-azure-upgrade
-  interval: 24h
+  cron: 53 12 */7 * *
   steps:
     cluster_profile: azure4
     env:
-      TEST_REQUIRES_SSH: "yes"
+      TEST_REQUIRES_SSH: "true"
```
Comment (jmguzik) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml:41]: make update reconciled

File: ci-operator/config/openshift/cluster-api-provider-ovirt/openshift-cluster-api-provider-ovirt-release-4.10.yaml (hunk 2 above)
Comment (jupierce) [reply ci-operator/config/openshift/cluster-api-provider-ovirt/openshift-cluster-api-provider-ovirt-release-4.10.yaml:59]: Yeah - I've seen this one in the analytics data. They insist it is OK and it certainly doesn't stand up clusters. Let's preserve it.

Comment (jmguzik) [reply ci-operator/config/openshift/cluster-api-provider-ovirt/openshift-cluster-api-provider-ovirt-release-4.10.yaml]: done, I kept also 4h interval in the job above as I feel is somehow connected. We can correct that in the 2nd run if needed.

**Pull request merged**